test/http.js

```diff
@@ -360,64 +359,55 @@ describe('http', () => {
     test(
       'should serialize fetch-like response and call serializeHeaders',
       () => {
-        const headers = {
-          Authorization: ['Basic hoop-la', 'Advanced hoop-la']
-        }
-
-        const res = fetchMock.mock('http://swagger.io', {headers})
+        const response = new Response('data', {
```

Comment:
I suspect `new Response` will require an eslint-disable. same reason as `new Headers` above

Reply:
Nope, I've made a global [override](https://github.com/swagger-api/swagger-js/pull/1523/files#diff-799dd7ade334108d9d2c335b978f7521R5). If it would require the build would fail as the linting is part of build process. Anyway I'll put the eslint disables on line basis instead of global override as discussed [here](https://github.com/swagger-api/swagger-js/pull/1523/files#r419903324)